packages/core/vite.config.ts

```diff
@@ -16,9 +16,14 @@ export default defineConfig({
   build: {
     sourcemap: true,
     lib: {
-      entry: path.resolve(__dirname, "src/index.ts"),
+      entry: {
+        blocknote: path.resolve(__dirname, "src/index.ts"),
+        comments: path.resolve(__dirname, "src/extensions/Comments/index.ts"),
+      },
       name: "blocknote",
-      fileName: "blocknote",
+      formats: ["es", "cjs"],
```

Comment:
should we apply this to all other packages as well? i.e: only es / cjs for all packages.

perhaps we can also streamline this further so we can re-use the vite config across projects, instead of having a duplicated file everywhere